### PR TITLE
[docs] DMS: Add yb_migrate documentation.

### PR DESCRIPTION
**Pull request by amit-yb**

### CODE REVIEW

**Comment by polarweasel** [comment]
@amit-yb do you have the diagram(s) available that are referenced in this doc?

File: docs/content/preview/migrate/yb_migrate.md [hunk 1]
```diff
@@ -0,0 +1,320 @@
+## Overview
+
+yb_migrate is an open-source database migration engine provided by YugabyteDB. It is a command line executable program that helps in easily migrating databases from PostgreSQL, Oracle, and MySQL to a YugabyteDB database.
```
Comment (polarweasel) [inline docs/content/preview/migrate/yb_migrate.md:3]: Add URL to the repo :)

File: docs/content/preview/migrate/yb_migrate.md [hunk 2]
```diff
@@ -0,0 +1,338 @@
+---
+title: Database Migration Service
+headerTitle: Database Migration Service
+linkTitle: Database Migration Service
```
Comment (aatrey-yb) [inline docs/content/preview/migrate/yb_migrate.md:4]: title: Database Migration Engine
headerTitle: Database Migration Engine
linkTitle: Database Migration Engine

**Comment by aishwarya24** [comment]
closing this issue as it was used for the first release of YugabyteDB-Voyager.

**Pull request closed without merging**